wof/examples/flask/odm_1_1/runserver_odm_1_1.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
```

Comment:
This one will always call on the system Python causing some grief to users trying to use `vens` or `conda`.